website/docs/docs/trusted-adapters.md

```diff
@@ -4,38 +4,22 @@ id: "trusted-adapters"
 hide_table_of_contents: true
 ---
 
-Trusted adapters are adapters not maintained by dbt Labs, that we feel comfortable recommending to users for use in production.
 
-Free and open-source tools for the data professional are increasingly abundant. This is by-and-large a *good thing*, however it requires due diligence that wasn't required in a paid-license, closed-source software world. As a user, there are questions to answer important before taking a dependency on an open-source project. The trusted adapter designation is meant to streamline this process for end users.
-
-<details><summary>Considerations for depending on an open-source project</summary>
-
-1. Does it work?
-2. Does anyone "own" the code, or is anyone liable for ensuring it works?
-3. Do bugs get fixed quickly?
-4. Does it stay up-to-date with new Core features?
-5. Is the usage substantial enough to self-sustain?
-pendency on this library?
+Trusted adapters take part in the Trusted Adapter Program, including a commitment to meet the program's requirements. They are maintained by dbt Labs, partners, and community members. Trusted adapters in dbt Cloud undergo an additional rigorous process that covers development, documentation, user experience, and maintenance requirements. We strongly recommend using them in production environments. For further details, refer to [What it means to be trusted](/guides/adapter-creation?step=8#what-it-means-to-be-trusted).
```

Comment:
leaning into the section title here instead of the guide title for the link. lemme know what you think!